src/VisualStudio/Core/Def/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -109,8 +109,8 @@ private abstract class AbstractTableDataSourceFindUsagesContext :
         /// </summary>
         private bool _currentlyGroupingByDefinition;
 
-        protected readonly List<Entry> EntriesWhenNotGroupingByDefinition = [];
-        protected readonly List<Entry> EntriesWhenGroupingByDefinition = [];
+        protected readonly (List<Entry> primary, List<Entry> nonPrimary) EntriesWhenNotGroupingByDefinition = ([], []);
+        protected readonly (List<Entry> primary, List<Entry> nonPrimary) EntriesWhenGroupingByDefinition = ([], []);
```

Comment:
general idea is that we have 'entries' sorted into two sections.  The 'primary' entries (those associated with the original symbol teh user is finding) and non-primary (those that came about from cascading).